### PR TITLE
removed last dynamics

### PR DESCRIPTION
**Pull request by kvish**
removed last dynamics from telemetryApiController

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kvish__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs [hunk 1]
```diff
@@ -335,8 +333,49 @@ public class TelemetryApiController : WebApiControllerBase
...
-                        deviceId = DeviceSchemaHelper.GetDeviceID(devInfo);
-                        props = DeviceSchemaHelper.GetDeviceProperties(devInfo);
+                        deviceId = devInfo.DeviceProperties.DeviceID;
+                        props = devInfo.DeviceProperties;
+                        enabledState = props.HubEnabledState;
+                    }
+                    catch (DeviceRequiredPropertyNotFoundException)
+                    {
+                        continue;
+                    }
```
Comment (handeadiguzel) [inline DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs]: so this exception: DeviceRequiredPropertyNotFoundException was thrown from: GetDeviceID

Looks like they're ignoring devices with null Ids.

Comment (kapil-ms) [inline DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs]: But now this exceptions won't be thrown, nullargumentexception would be thrown or something similar in C#

**Comment by kapil-ms** [comment]
lgtm

**Pull request merged**